server/plugins/task-plugin-api/src/main/java/com/oceanbase/odc/plugin/task/api/datatransfer/model/ObjectResult.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.odc.plugin.task.api.datatransfer.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.oceanbase.tools.loaddump.common.model.ObjectStatus;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class ObjectResult extends ObjectStatus {
+
+    /**
+     * for export only, internal usage
+     */
+    @JsonIgnore
+    private String[] exportPaths;
```

Comment:
use outputstream instead

Reply:
use URL instead of stream, because service need it to delete export files.